### DISCUSSION TITLE
Goal 3: Ensure healthy lives and promote well-being for all at all ages

### DISCUSSION THREAD

**Discussion by HSCunha** [Project-Resilience/platform#25] [General]
Chat about anything related to Goal 3: Ensure healthy lives and promote well-being for all at all ages

**Comment by danyoungday** [comment]
The MIMIC dataset by MIT records medical data and decisions made in the hospital: [https://mimic.mit.edu/](https://mimic.mit.edu/)
It requires an account and a request to access it.

This could be a time-series problem where a patient's status and then the action taken at each individual time step is taken into account, with the outcomes being their eventual condition and some sort of cost metric.

There is the additional concern that records may be too unique to format into nice, tabular data for something like NeuroAI.

**Comment by GPaolo** [comment]
Me and Olivier have made an adaptation for of TerraLingua to simulate the Ebola pandemic.
It can be very useful to test different communication strategies and sickness prevention methods.

WHO published recently a new set of recommendations for fighting against Ebola: https://www.who.int/news/item/24-08-2026-second-meeting-of-the-ihr-emergency-committee-on-the-epidemic-of-ebola-bundibugyo-virus-disease-in-the-democratic-republic-of-the-congo-temporary-recommendations
We have tried to include some of them into the simulation, mainly the community engagement, given that it seems to be a critical aspect of the fight against the virus https://www.who.int/news/item/06-08-2026-africa-cdc-and-who-call-for-urgent--community-led-action-to-contain-ebola-in-the-drc

The way the virus spreads and kills people has been modeled following the information found here: https://www.who.int/news-room/fact-sheets/detail/ebola-disease

The code is here: https://github.com/GPaolo/terralingua-pandemics
It can simulate different personas (medics, who operators, religious leaders, traditional healers etc)
There is also a system that can help analyse the results and create plots regarding the simulation.

And here there is a tool that you can install in TL to help you setup the run parameters and launch the simulation: https://github.com/GPaolo/terralingua_launcher

(they are both in active development, so things will change and improve with time).
